_includes/specials/taproot/en/05-taproot-notebooks.md

```diff
@@ -0,0 +1,55 @@
+Almost two years ago, James Chiang and Elichai Turkel produced an [open source
+repository][taproot-workshop] of Jupyter notebooks for a series of
+Optech workshops to train developers on [taproot][topic taproot] technology.
+Workshops [held][workshops] in San Francisco, New York City, and London
+received positive reviews, but travel restrictions prevented subsequent
+in-person workshops.
+
+Since the publication of the Jupyter notebooks, taproot underwent
+several changes.  However, taproot support was also merged into Bitcoin
+Core, allowing the notebooks to drop their dependency on a custom branch
+of Bitcoin Core.  Developer Elle Mouton has kindly [updated][mouton
+tweet] the [notebooks][notebooks #168] for all those changes, making
+them again a great way to quickly build hands-on experience working with
+taproot's algorithms and data types.
+
+The notebooks are divided into four sections:
+
+- **Section 0** contains a notebook that helps you set up your
+  environment, covers the basics of elliptic curve cryptography, and
+  teaches you about the tagged hashes used throughout BIPs
+  [340][BIP340], [341][BIP341], and [342][BIP342].
+
+- **Section 1** walks you through creating [schnorr signatures][topic
+  schnorr signatures].  Once you've mastered them, you learn how to
+  create [multisignatures][topic multisignature] with the [MuSig][topic
+  musig] protocol.
+
+- **Section 2** gives you experience with every aspect of taproot.  It
+  starts with a review of the principles of segwit v0 transactions and then
+  helps you create and spend segwit v1 (taproot) transactions.  Applying
```

Comment:
Instead of this I did s/spend/send/ since the section being described also includes other bits about serializing the transaction and creating the signature hash.